### PR TITLE
fix(ui-toolkit): toolkit bugs

### PR DESCRIPTION
**Pull request by dexturr**
# Related issues 🔗

Minor UI toolit bugs

# Description ℹ️

- Has error did not highlight border & text with correct colors
- Checkboxes were centre aligned instead of aligned at start when overflowing
- Secondary buttons should always have white text
- Ternary buttons should always have black text

### CODE REVIEW

File: libs/ui-toolkit/src/components/button/button.tsx [hunk 1]
```diff
@@ -30,14 +30,14 @@ const primary = [
   'enabled:active:bg-vega-yellow-550 enabled:active:border-vega-yellow-550',
 ];
 const secondary = [
-  'text-white dark:text-black',
```
Comment (MadalinaRaicu) [inline libs/ui-toolkit/src/components/button/button.tsx:33]: ahm, are you sure this works fine on every theme?

Comment (dexturr) [reply libs/ui-toolkit/src/components/button/button.tsx:33]: As per designs:

Screenshot 2023-07-14 at 12 44 32
ge per theme np?

Comment (dexturr) [reply libs/ui-toolkit/src/components/button/button.tsx:33]: (Secondary is pink)

Comment (MadalinaRaicu) [reply libs/ui-toolkit/src/components/button/button.tsx:33]: alright

**Review by MadalinaRaicu — APPROVED** [review]

**Pull request merged**